### DISCUSSION TITLE
TumblThree latest versions doesn't work on windows 7

### DISCUSSION THREAD

**Discussion by kechen1122** [TumblThreeApp/TumblThree#362] [Q&A (Question & Answer)]
Hello, I've been using this wonderful app for over a year and when I tried to get back to it with the latest version 1.41, it seems this version (or the previous ones like 1.30, 1.31) don't work on my windows 7 pc anymore. There were no errors popup whenever I try to run the app so I really don't know what the cause is. I have no problems using the older versions but it looks like they also couldn't download images properly like they did for me before. The oldest version that I used is this v1.0.10.1008. The app opened just fine but it only downloaded broken images after successfully crawling.

**Comment by thomas694** [comment]
Hello,
in general TumblThree runs under Windows 7. Sometimes users initially report that they have a problem starting or running it. But until now we haven't got an answer from them, how they solved it.

So we need more information from you and here are a few things you can check.
Do you have any antivirus/security/firewall software installed which could prevent executing the program? Can you please check the windows event logs for a hint what happens when you try to start the exe?
As the program doesn't even start I think our log file (folder %LOCALAPPDATA%\TumblThree) won't have any hints.
Is your user account an administrator or normal user?

Do you have the same problem with other programs/exe too?
Have you already tried to create a new windows user account and run the exe under that one?

**Comment by thomas694** [comment]
As mentioned in #151 you could delete (don't forget a backup) or rename the settings folder (%LOCALAPPDATA%\TumblThree) to see if the new version starts afterwards.

**Comment by vozmor** [comment]
Thank you very much for your help, @thomas694 and @Fehertukor. I succesfully narrowed the problem to the **Setting.json** file in AppData/Local/TumblThree/Settings. Now it works beautifully. Thank you once again for this wonderful app.

**Comment by thomas694** [comment]
@kechen1122 
Please try the newest version 1.5.2 and let us know, if it works for you.

**Comment by thomas694** [comment]
The issue has been closed. You can still comment. Feel free to ask for reopening the issue if needed.
